packages/dashboard/src/containers/App.js

```diff
@@ -1,27 +1,105 @@
-import React from 'react'
-import { HashRouter as Router, Switch, Route } from 'react-router-dom'
+import React, { useState, useEffect } from 'react'
+import { HashRouter as Router, Switch, Route, Redirect } from 'react-router-dom'
 import { SocketIOProvider } from 'use-socketio'
+import { Transition, TransitionGroup } from 'react-transition-group'
 
 import { ThemeProvider, unstable_createMuiStrictModeTheme as createMuiTheme } from '@material-ui/core'
+import CssBaseline from '@material-ui/core/CssBaseline'
 
 import Layout from '../components/Layout'
 
 import { AppStateProvider } from '../context/app-state'
 
 import Dashboard from './Dashboard'
+import SplashScreen from '../components/SplashScreen'
+import NoMatch from '../components/NoMatch'
+import { play, exit } from '../timeline'
```

Comment:
All good, just import sort.
```suggestion
import { play, exit } from '../timeline'

import { AppStateProvider } from '../context/app-state'

import Layout from '../components/Layout'
import NoMatch from '../components/NoMatch'
import SplashScreen from '../components/SplashScreen'

import Dashboard from './Dashboard'
```